docs/guide/build.md

```diff
@@ -224,11 +224,11 @@ If the hashed assets and public files aren't deployed together, options for each
 
 ```js
 experimental: {
-  renderBuiltUrl(filename: string, { hostType: 'js' | 'css' | 'html', type: 'public' | 'asset' }) {
+  renderBuiltUrl(filename: string, { hostId: string, hostType: 'js' | 'css' | 'html', type: 'public' | 'asset' }) {
```

Comment:
Thanks for the fix, you made me remember we have a bug here, this isn't proper typescript. Would you mind correcting that too? 
```suggestion
  renderBuiltUrl(filename: string, { hostId, hostType, type }: { hostId: string, hostType: 'js' | 'css' | 'html', type: 'public' | 'asset' })) {
```
And it should also be ```ts instead of js